tests/test_plugin.py

```diff
@@ -1200,3 +1200,18 @@ def test_replacement_payload(node_factory):
         l1.rpc.pay(inv)
 
     assert l2.daemon.wait_for_log("Attept to pay.*with wrong secret")
+
+
+def test_plugin_fail(node_factory):
+    """Test that a plugin which fails (not during a command)"""
+    plugin = os.path.join(os.path.dirname(__file__), 'plugins/fail_by_itself.py')
+    l1 = node_factory.get_node(options={"plugin": plugin})
+
```

Comment:
We might want to check that `failcmd` is in help before the plugin spontaneously catches fire :-)

Reply:
Timing issue though, so I didn't bother...